package.json

```diff
@@ -64,7 +64,7 @@
     "decompress-zip": "^0.3.0",
     "electron-log": "^2.2.6",
     "electron-store": "^1.1.0",
-    "emerald-js": "git+https://github.com/ethereumproject/emerald-js.git#v0.0.11",
+    "emerald-js": "git+https://github.com/shanejonas/emerald-js.git#feature/unhide-accounts",
```

Comment:
[WIP] change this when `emerald-js` is merged

Reply:
I'v tagged emerald-js v0.0.12, it is possible to fix dependency